bool real_d435
sensor_msgs/PointCloud2 scene_cloud
sensor_msgs/Image scene_depth_img
sensor_msgs/Image scene_color_img
string scene_cloud_save_path
string scene_depth_image_save_path
string scene_color_image_save_path
---
bool save_visual_data_success
